interfaces/builtin/kerberos_tickets.go

```diff
@@ -0,0 +1,44 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const krbSummary = `allows reading to Kerberos tickets in /tmp.`
+
+const krbBaseDeclarationSlots = `
+  kerberos-tickets:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const krbConnectedPlugAppArmor = `
+/var/lib/snapd/hostfs/tmp/krb5cc* r,
```

Comment:
How does the DIR: case work in this manner? I think we'd need more permissions for it to do something.

Can you provide an example of how real-world DIR: value might look like?

Reply:
`DIR:/tmp` will cause ticket files to be created in `/tmp`, so access to `/tmp/krb5cc*` would still be enough.

Reply:
I also had to add the lock permission.